packages/nextjs/components/scaffold-eth/Contract/InheritanceTooltip.tsx

```diff
@@ -0,0 +1,14 @@
+import { InformationCircleIcon } from "@heroicons/react/24/solid";
+
+export const InheritanceTooltip = ({ inheritedFrom }: { inheritedFrom?: string }) => (
+  <>
+    {inheritedFrom && (
+      <span
+        className={`cursor-help tooltip tooltip-top tooltip-accent px-2 before:content-[attr(data-tip)]`}
+        data-tip={`Inherited from: ${inheritedFrom}`}
+      >
+        <InformationCircleIcon className="h-4 w-4 text-gray-400" aria-hidden="true" />
```

Comment:
NITPICK: Maybe we can remove `text-gray-400` ?

Reply:
I thought it looked better with it, visually. But I'm all for keeping the code simple :smile:

Reply:
Fixed in: https://github.com/FilipHarald/scaffold-eth-2-sandbox/commit/a4e3e42a3a058c3b4129bdc5b881a82596d56ffa